department-of-veterans-affairs/content-build
# Update/unit test process from vets website

Updated dependencies to get unit tests passing via local `yarn test:unit`

**Validation Instructions**

1. Pull down the branch
2. Run `rm -rf node_modules package-lock.json yarn.lock && yarn` to reset the environment
3. Run `yarn test:unit`

All 65 tests should pass.
